build/alpine/Dockerfile.prebuilder

```diff
@@ -34,7 +34,7 @@ RUN apk add --no-cache \
         snappy-dev \
         llvm-libunwind-dev \
         bash \
-        xz \
+        xz-dev \
```

Comment:
I think we need `xz` for `ghcup` to work.

Reply:
I thought one included the other, apparently not. Thanks for catching this issue, I now include both.